packages/atlas/src/views/studio/StudioWelcomeView/StudioWelcomeView.styles.ts

```diff
@@ -48,24 +95,200 @@ export const StyledSignInIllustrationSVG = styled(SvgSigninIllustration)`
   }
 `
 
+export const StepsContainer = styled.div`
+  display: flex;
+  width: 100%;
+  position: relative;
+  padding: ${sizes(1)} 0;
+  align-items: center;
+  margin: ${sizes(6)} 0;
+  box-shadow: inset 0 1px 0 0 ${cVar('colorBorderMutedAlpha')}, inset 0 -1px 0 0 ${cVar('colorBorderMutedAlpha')};
+  ${media.md} {
+    margin: ${sizes(8)} 0;
+    overflow: hidden;
+    padding: ${sizes(3)};
+
+    ::before,
+    ::after {
+      position: absolute;
+      content: '';
+      background: linear-gradient(
+        90deg,
+        #000 0%,
+        #000 0.75%,
+        rgb(0 0 0 / 0.9976) 2.92%,
+        rgb(0 0 0 / 0.992) 6.37%,
+        rgb(0 0 0 / 0.981) 10.97%,
+        rgb(0 0 0 / 0.963) 16.59%,
+        rgb(0 0 0 / 0.936) 23.1%,
+        rgb(0 0 0 / 0.8984) 30.37%,
+        rgb(0 0 0 / 0.8483) 38.27%,
+        rgb(0 0 0 / 0.784) 46.66%,
+        rgb(0 0 0 / 0.7037) 55.41%,
+        rgb(0 0 0 / 0.6056) 64.39%,
+        rgb(0 0 0 / 0.488) 73.47%,
+        rgb(0 0 0 / 0.349) 82.52%,
+        rgb(0 0 0 / 0.187) 91.41%,
+        rgb(0 0 0 / 0) 100%
+      );
+      width: 24px;
+      z-index: 1;
```

Comment:
Can we use some value from `zIndex` const?

Reply:
I don't agree. Not everything should use const, this is used only locally - to put a shadow upfront